instance/management/commands/migrate_ephemeral.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2015-2016 OpenCraft <contact@opencraft.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+    This management command will migrate instances using ephemeral databases
+    to external databases instead.
+
+    This is intended to deprecate ephemeral databases.
```

Comment:
@viadanna Thanks for turning the script into a management command!

Two things:

* Let's be even more explicit here and say something like:
    > This is a prerequisite for deprecating ephemeral databases, cf. https://github.com/open-craft/opencraft/pull/327.
* Nit: Any specific reason for indenting this text? I think it would make sense to remove the indentation to be consistent with the formatting of other module-level docstrings in this repo.